src/findReferences.ts

```diff
@@ -69,4 +71,12 @@ export const registerFindReferences = () => {
         activeEditor.selections = [new vscode.Selection(range.end, range.end)]
         await vscode.commands.executeCommand('editor.action.referenceSearch.trigger')
     })
+    const setReferenceButtonVisibility = () => {
```

Comment:
You removed initial call of `setReferenceButtonVisibility`. Everything else looks good.